charts/dave/charts/backend/values.yaml

```
@@ -100,6 +100,9 @@ extraEnvVars:
 #  - name: "TZ"
 #    value: "Europe/Berlin"
 
+backend:
+  existingSecret: ""
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)